__tests__/article-test.js

```diff
@@ -43,3 +43,33 @@ describe.only('Get article by ID', () => {
     });
 });
 
+describe('Get articles', () => {
+    it('Returns correct number of articles', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles).toHaveLength(5);
+            });
+    });
+    it('Returns articles in the correct format', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                body.articles.forEach((article) => {
+                    expect(article).toMatchObject({
+                        article_id: expect.any(Number),
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String),
+                        comment_count: expect.any(Number)
+                    })
+                });
+            });
+    });
```

Comment:
We should write a test to check that the articles can be sorted as mentioned in the instructions 🙂